DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -16,7 +16,9 @@
   <class name="pat::Lepton<reco::BaseTau>" />
 
   <!-- PAT Objects, and embedded data  -->
-  <class name="pat::Electron"  ClassVersion="33">
+  <class name="pat::Electron"  ClassVersion="35">
+   <version ClassVersion="35" checksum="1857282274"/>
+   <version ClassVersion="34" checksum="3659401979"/>
```

Comment:
I cannot compile with 34 only. I get an error:
error: class 'pat::Electron' has a different checksum for ClassVersion 34. Increment ClassVersion to 35 and assign it to checksum 1857282274
Suggestion: You can run 'scram build updateclassversion' to generate src/DataFormats/PatCandidates/src/classes_def_objects.xml.generated with updated ClassVersion